tests/functional/test_inference_per_framework.py

```diff
@@ -1136,6 +1136,34 @@ def test_chat_api(self, nim_predictor, streaming, nchoices):
 
         assert "Boston! One of the oldest and most historic cities" in llm_response
 
+    @pytest.mark.parametrize("path", ["directAccess", "nim"])
+    def test_forward_http_request(self, path, nim_predictor):
```

Comment:
This test is testing two things. In theory you should split into two test functions `test_forward_http_request_get` and `test_forward_http_request_post`